src/entity/Guild/GuildUserBan.ts

```diff
@@ -0,0 +1,34 @@
+import {
+  Entity,
+  Column,
+  JoinColumn,
+  Index,
+  ManyToOne,
+  PrimaryGeneratedColumn
+} from 'typeorm'
+import { GuildUser } from './GuildUser'
+
+@Entity()
+export class GuildUserBan {
+  @PrimaryGeneratedColumn() id: number
+
+  @Column('varchar') issuerId: string
+
+  @Column('timestamp without time zone') timestamp: Date
+
+  @Column('varchar') reason: string
+
+  @Column('boolean') unbanned: boolean
```

Comment:
Remove this field and replace it with a nullable string column called `length`

This will be used to determine if the ban was temporary or permanent.

It may be null, or it may be like `3d` or `1mo` or `15m`. Short strings like that.
Also give the column a limit of 10 characters.